common/app/views/fragments/nav/editionPicker.scala.html

```diff
@@ -0,0 +1,19 @@
+@()(implicit request: RequestHeader)
+
+@import common.{Edition, LinkTo}
+
+<div class="new-header__edition-picker">
+    @fragments.inlineSvg("world", "icon")
+
+    <select onchange="location = this.value;">
```

Comment:
@regiskuckaertz 
> This won't work via keyboard arrows (I think) because the change event is triggered every time the user activates an option, and in order to activate the third option she has to go through the first and the second.

This seems to work with keyboard arrows and you have to press enter to activate the change event. That said, I have only tested on Chrome so far. I will so some cross browser testing today